packages/helix-shared-config/package.json

```diff
@@ -30,6 +30,7 @@
     "ajv": "8.6.2",
     "ajv-formats": "2.1.1",
     "fs-extra": "10.0.0",
+    "ignore": "^5.1.8",
```

Comment:
nitpicky: make it a habit of adding it exact. (`-E`). otherwise renovate will make an extra PR.